### PR TITLE
Build column description in the plugin

### PR DESCRIPTION
**Pull request by kilnerm**
This PR updates the plugin to conform to the new QueryBuilder requirements and adds the logic to build a string representation of a column for the plugin.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -29,7 +29,8 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/IBM-Swift/CLibpq.git", .upToNextMinor(from: "0.1.0")),
-        .package(url: "https://github.com/IBM-Swift/Swift-Kuery.git", from: "2.0.0"),
+        //.package(url: "https://github.com/IBM-Swift/Swift-Kuery.git", from: "2.0.0"),
+        .package(url: "https://github.com/IBM-Swift/Swift-Kuery.git", .branch("issue_column_attributes") ),
```
Comment (ddunn2) [inline Package.swift]: ToDo: Remember to revert before merging.

File: Sources/SwiftKueryPostgreSQL/PostgreSQLConnection.swift [hunk 2]
```diff
@@ -586,3 +572,89 @@ public class PostgreSQLConnection: Connection {
         return nil
     }
 }
+
+class PostgreSQLColumnBuilder: ColumnCreator {
+    func buildColumn(for column: Column, using queryBuilder: QueryBuilder) -> String? {
+        guard let type = column.type else {
+            return nil
```
Comment (ddunn2) [inline Sources/SwiftKueryPostgreSQL/PostgreSQLConnection.swift]: Is it possible for error handling here?

I'm not sure under what circumstances column.type could be nil, but is it something we should alert a user to?

File: Sources/SwiftKueryPostgreSQL/PostgreSQLConnection.swift [hunk 3]
```diff
@@ -586,3 +572,89 @@ public class PostgreSQLConnection: Connection {
...
+        }
+
+        var typeString = type.create(queryBuilder: queryBuilder)
+        if let length = column.length {
+            typeString += "(\(length))"
+        }
+        if column.autoIncrement {
+            guard let autoIncrementType = getAutoIncrementType(for: typeString) else {
+                //Unrecognised type for autoIncrement column, return nil
+                return nil
```
Comment (ddunn2) [inline Sources/SwiftKueryPostgreSQL/PostgreSQLConnection.swift]: Same as above.

I'm not sure how important it'd be to alert a user to the reason why this has returned `nil`

File: Sources/SwiftKueryPostgreSQL/PostgreSQLConnection.swift [hunk 4]
```diff
@@ -586,3 +572,89 @@ public class PostgreSQLConnection: Connection {
...
+        }
+        if column.isUnique {
+            result += " UNIQUE"
+        }
+        if let defaultValue = column.defaultValue {
+            var packedType: String
+            do {
+                packedType = try packType(defaultValue, queryBuilder: queryBuilder)
+            } catch {
+                return nil
```
Comment (ddunn2) [inline Sources/SwiftKueryPostgreSQL/PostgreSQLConnection.swift]: Same as above.

File: Sources/SwiftKueryPostgreSQL/PostgreSQLConnection.swift (hunk 2 above)
Comment (kilnerm) [reply Sources/SwiftKueryPostgreSQL/PostgreSQLConnection.swift]: The nil is handled in the calling function and an error thrown that identifies the column as having invalid attributes.

File: Sources/SwiftKueryPostgreSQL/PostgreSQLConnection.swift (hunk 3 above)
Comment (kilnerm) [reply Sources/SwiftKueryPostgreSQL/PostgreSQLConnection.swift]: The nil is handled in the calling function and an error thrown that identifies the column as having invalid attributes.

File: Sources/SwiftKueryPostgreSQL/PostgreSQLConnection.swift (hunk 4 above)
Comment (kilnerm) [reply Sources/SwiftKueryPostgreSQL/PostgreSQLConnection.swift]: The nil is handled in the calling function and an error thrown that identifies the column as having invalid attributes.

**Pull request merged**